doc/rfcs/004-quickly-collaborate-with-coworkers-and-friends.md

```diff
@@ -0,0 +1,166 @@
+# Quickly collaborate with coworkers and friends
+
+## Status
+
+Proposed
+
+## Summary
+
+A user can add any of their recent collaborators to a list of "trusted collaborators." Once two people have added each other to their list of trusted collaborators, they can establish future collaboration sessions with each other from directly within Atom.
+
+## Motivation
+
+We hope to encourage more collaboration by reducing the barriers to entry. Specifically, we want to reduce the number of steps that exist between A) deciding that you *want* to collaborate and B) *actually* collaborating.
+
+Today, the transition from A to B involves the following steps:
+
+1. Host shares a portal
+2. Host copies portal URL to their clipboard
+3. Host switches from Atom to third-party communication tool (e.g., Slack, IRC)
+4. Host pastes portal URL into third-party communication tool
+5. Guest clicks portal URL, and the operating system hands control to Atom, and Atom joins the portal
+
+With the ability to invite trusted collaborators to your portal from within Atom, we reduce the set-up process from 5 steps to just 3 steps:
+
+1. Host selects a person in their list of trusted collaborators
+2. Host invites the selected person to a portal, and Atom automatically shares a portal for the host and invites the selected person
+3. Guest accepts invitation to join portal, and Atom joins the portal
+
+## Explanation
+
+### Accept portal invitations from a trusted collaborator
+
+Teletype provides a list of your recent collaborators. Each time a guest joins your portal, Teletype adds them to your list of recent collaborators. Each time you join a portal, Teletype adds the host to your list of recent collaborators.
```

Comment:
Can we make sure that we do not lose the ability to remove recent collaborators? Removing trusted collaborators is noted before, but it would be great to also note how to remove recent collaborators!

Reply:
@jasonrudolph  Sure! Let's say I accidentally click on a link in Slack and realize it's someone I don't actually like that's hosting the Teletype portal. That person then shows up in my recent collaborators list. It's not explicitly clear to me if/when their username will disappear from my recent collaborators list, so I would like the ability to click an (x) to remove them from the list. Does that make sense?

Reply:
@dmleong: Thanks for that context. :zap::bow: 

46c56136ab02dd70f707726896000e0ba3928c82 notes that Teletype should provide the ability to remove anyone from your list of past collaborators.

Reply:
If the user opens up new Atom window without any recent (or "favorite") projects loaded, a possible solution may be to have a "Recent Projects" tab holds a menu of the user's recent projects. After clicking on the menu item for "Project X" (for instance) the view will then transition from a menu of projects, to a list of recent collaborators on Project X.